Azure-Samples/ctfd-azure-paas
# Replace MariaDB with MySQL

## Purpose
MariaDB is no longer supported in Azure. This PR replaces the implementation with a MySQL server

Fixing #18 


## Does this introduce a breaking change?

```
[x] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
* Deploy to Azure (with and without vnet)
* Check the deployment is working by browsing to the CTFD webapp site
